docs/career_assessment_how_to.md

```diff
@@ -0,0 +1,65 @@
+If you're setting up a new copy of this site, use this google spreadsheet template: https://drive.google.com/previewtemplate?id=1vgqsoXLJ9FSDZamGv-SzDX_xCV3wHY0Lh142nhuZDQI&mode=public. After filling in the info, go to 'File' > 'Publish to Web' and publish it as a web page. Make sure "Automatically republish when changes are made" is checked at the bottom. Save the link! You can also get the link again by opening the "Publish to Web" dialog again.
+
+### Updating career info
+To update the information about a career: edit your spreadsheet, then go to the ["manage" page](http://workforwardnola.com/manage) & enter the username and password. Enter the PUBLIC link to the spreadsheet (it ends in `/pubhtml`) (Like https://docs.google.com/spreadsheets/d/1lMeZZ_vu-xIVKLgo7obKlF0X4iJ3oHkLKl8uIrPsSt8/pubhtml), wait for the link to be validated, then click "Update content".
+
+### Adding new career
+Add a new row to the google spreadsheet. The columns `name`, `sector`, and `experienced_wage` are required. The `traits` column is required to have the new career be included in assessment results. After filling in the new row in the spreadsheet, follow the instructions above to load the new information.
+
+## Adding new trait
+Adding a new trait to the assessment cannot be done without additional coding work.
+
+1. Add the new trait(s) to the "traits" tab of the spreadsheet with an `id`. Add that `id` to the `traits` column of any career you wish to associate it with.
+2. In `views/assessment.rb` the trait is associated with an image, other traits as a "compound trait", or as a "reverse trait"--the opposite of an image. The name of the trait must be entered somewhere in this file or it will not be part of the assessment.
+3. Assessment images are saved in `/assets/images/assessment`
+
+### Assessment pictures <-> traits
+"Standard" traits:
+If a user answers 'me', the `trait` is recorded, if the answer is 'not me', the `reverse_trait` is recorded.
+```ruby
+{ trait: 'Following instructions',    # must match name of trait from spreadsheet/database
+  reverse_trait: 'Self-starter',      # optional, is 'inverse' of trait above
+  title: 'Following instructions',    # text to display with image
+  alt: 'Person is looking at a recipe on a computer before they begin chopping vegetables',   # text describing image, for accessibility
+  file: 'instructions.jpg' },         # file name of image (in /assets/images/assessment)
+```
+"Compound" traits:
+A trait may be tricky to capture in a single image. Two or more _existing_ traits can be combined to form a compound trait. If a user checks 'me' for each of the components, a 'me' is recorded for the trait.
+```ruby
+{ trait: 'Perceptive',    # trait name
+  components: ['Detail oriented', 'Customer service'] },  # names of simpler traits
+```
+
+## Adding or removing a column from the careers spreadsheet
+If the case is you no longer want to show a specific piece of information, it can be left out of the HTML.
```

Comment:
Wording is a little funky here. 

Perhaps: To omit a specific piece of existing information, 'comment out' or delete it from the HTML.